radio/src/targets/common/arm/stm32/adc_driver.cpp

```diff
@@ -24,10 +24,12 @@
   // not needed
 #elif defined(PCBX10)
   const int8_t ana_direction[NUM_ANALOGS] = {1,-1,1,-1,  -1,-1,-1, -1,1,1,1,  -1};
-#elif defined(PCBX9E) && defined(HORUS_STICKS)
-  const int8_t ana_direction[NUM_ANALOGS] = {1,-1,1,-1,  -1,-1,-1,1, -1,1,1,1,  -1};
 #elif defined(PCBX9E)
-  const int8_t ana_direction[NUM_ANALOGS] = {1,1,-1,-1,  -1,-1,-1,1, -1,1,1,1,  -1};
+  #if defined(HORUS_STICKS)
```

Comment:
Not too late to start though... 😉   It is done in a few places already, but not consistently.  I find it helps a lot in some cases, very awkward-looking in others.  The common problem with indenting them around C code (like this) is which level to indent the C code to... the # directives or the other C code.